Dockerfile.app

```diff
@@ -3,26 +3,25 @@ ARG NPM_VERSION="5.5.0"
 FROM registry.devshift.net/fabric8-hdd/nodejs:${NODE_VERSION}_npm_${NPM_VERSION}
 LABEL maintainer="Anmol Babu <anmolbudugutta@gmail.com>"
 
-# Setup App Dir
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
-
 # Install git for npm to be able to pull dependencies from git
 RUN yum -y install git
 
 # Install app dependencies
 ADD package.json ./package.json
 ADD package-lock.json ./package-lock.json
-
-# Install app
 RUN npm install --production
 
+# Setup App Dir
+RUN mkdir -p /usr/src/app && cp -a ./node_modules /usr/src/app/
+WORKDIR /usr/src/app
+
 # Bundle app source
 ADD . .
```

Comment:
Now, the package.json and package-lock.json files are present in  / directory as well in /usr/src/app directory.

Can we ensure a clean up for  /package.json and package-lock.json ?

Reply:
Sure

Reply:
Really required?
They are just text files..